src/functions/accent.js

```diff
@@ -87,6 +89,9 @@ const htmlBuilder = (group, options) => {
             // shift the accent over to a place we don't want.
             accent.italic = 0;
             width = accent.width;
+            if (group.value.label === "\\c") {
+                clearance += accent.depth;
+            }
```

Comment:
I'm guessing `accent.depth` is zero for the other accents?  Still, might make sense to keep this wrapped in an `if` (possibly `if (accentBelow)`).